10 Ditto/10Ditto.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+
+    <script>
+    let obj1 = {thing : "hi",
+            Game : "Neighbor",
+            pokemon : "peecachu"}
+
+    let obj2 = {thing : "Neighbor",
+            Game : "Neighbor",
+            pokemon : "peecachu"}
+
+    let array = Object.keys(obj1);
```

Comment:
These can be `const` as well. I would make the variable names more descriptive in this case. Maybe something like `obj1Keys`